.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -29,6 +29,11 @@ If applicable, submit logs or stack traces from the affected services
 **System Information (please complete the following information):**
  - OS/Version: [e.g. RHEL 7.6]
  - Kubernetes Version [e.g. 1.18]
+ - karavi-topology Version: [e.g. 0.1.0-pre-release]
+ - Grafana Version: [e.g. 7.3.2]
+ - Grafana Dashboard Title: [e.g. Kubernetes Admin Dashboard]
+ - Grafana Datatable Panel plugin configuration (can be provided as a screenshot)
+ - Grafana JSON Datasource configuration (can be provided as a screenshot)
```

Comment:
JSON could be renamed to SimpleJson to match [getting started guide](https://github.com/dell/karavi-topology/blob/main/docs/GETTING_STARTED_GUIDE.md#configure-plugins).

Reply:
fixed.